src/app/hooks.ts

```diff
@@ -0,0 +1,5 @@
+import { useDispatch, useSelector } from 'react-redux';
+import { AppDispatch, RootState } from './store';
+
+export const useAppSelector = useSelector.withTypes<RootState>();
+export const useAppDispatch = useDispatch.withTypes<AppDispatch>();
```

Comment:
The `useDispatch` hook should be typed using TypeScript generics, like this: `export const useAppDispatch = () => useDispatch<AppDispatch>();`. The `withTypes` method does not exist.